app/tests/lectures/test_models.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+from django.db import IntegrityError
+from django.test import Client
+from lectures.models import Lecture
+from lectures.serializers import LectureSerializer
+from rest_framework.reverse import reverse
+
+
+@pytest.mark.django_db
+def test_create_lecture(create_valid_subject, create_valid_user):
+    """
+    Ensure we can create a new lecture.
+    """
+    data = {
+        "title": "Lecture",
+        "text": "Text",
+        "user": create_valid_user,
+        "subject": create_valid_subject,
+    }
+    lec = Lecture.objects.create(**data)
+    assert lec.title == "Lecture"
+    assert lec.text == "Text"
+
+
+@pytest.mark.django_db
+@pytest.mark.xfail(raises=IntegrityError)
+def test_create_lecture_without_user(create_valid_subject):
+    """
+    Ensure we can't create a new lecture without user.
+    """
+    data = {"title": "Lecture", "text": "Text", "subject": create_valid_subject}
+    Lecture.objects.create(**data)
+
+
+@pytest.mark.django_db
+@pytest.mark.xfail(raises=IntegrityError)
+def test_create_lecture_without_subject(create_valid_user):
+    """
+    Ensure we can't create a new lecture without subject.
+    """
+    data = {"title": "Lecture", "text": "Text", "user": create_valid_user}
+    Lecture.objects.create(**data)
+
+
+@pytest.mark.django_db
+def test_get_all_lectures(create_valid_lecture, create_second_valid_lecture):
+    """
+    Ensure we get all lectures in proper format.
+    """
+    create_valid_lecture
+    create_second_valid_lecture
+    client = Client()
```

Comment:
Do not create api client for each test, pytest-django has a build-in fixture client, that can be used like this:
```
@pytest.mark.django_db
def test_view(client):
```
There is no need to create a separate fixture for client, just use a built-in one.